docs/TestingApproach.md

```diff
@@ -5,6 +5,7 @@ Below we describe the set of functional, regression and cross-language tests tha
 * [Regression Tests](#regression-tests)
 * [Multi-threaded Tests](#multi-threaded-tests)
 * [Cross Language Testing Framework (CLTF)](#cross-language-testing-framework-cltf)
+* [Server-Client Tests]()
```

Comment:
i don't think this is necessary. Client's aren't part of the server feature, like reference apps.